src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -187,20 +187,20 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
                         OutputMessage.Severity.Warning));
                 }
 
-                ITransport localTransport = null;
+                ITransport localTransport = new VsCodeTerminalTransport();
                 // For local Linux and OS X launch, use the local Unix transport which creates a new terminal and
                 // uses fifos for debugger (e.g., gdb) communication.
                 if (this.MICommandFactory.UseExternalConsoleForLocalLaunch(localLaunchOptions) &&
                     (PlatformUtilities.IsLinux() || (PlatformUtilities.IsOSX() && localLaunchOptions.DebuggerMIMode != MIMode.Lldb)))
                 {
-                    localTransport = new LocalUnixTerminalTransport();
+                    //localTransport = new LocalUnixTerminalTransport();
 
                     // Only need to clear terminal for Linux and OS X local launch
                     _needTerminalReset = (!localLaunchOptions.ProcessId.HasValue && _launchOptions.DebuggerMIMode == MIMode.Gdb);
```

Comment:
>_needTerminalReset [](start = 20, length = 18)

Do we still need this

Reply:
Sort of. If we want to keep the original behavior when RunInTerminal isn't supported, then yes?